src/collectors/block_collector.rs

```diff
@@ -6,11 +6,11 @@ use ethers::{
     providers::JsonRpcClient,
     types::{BlockNumber, H256, U256, U64},
 };
-use tokio::time::sleep;
 use std::{sync::Arc, time::Duration};
+use tokio::time::sleep;
 use tracing::{error, info, warn};
 
-const BLOCK_POLLING_INTERVAL: Duration = Duration::from_millis(200);
+const BLOCK_POLLING_INTERVAL: Duration = Duration::from_millis(250);
```

Comment:
I think it's fine since it was 200 before for Mainnet + Base.. I just bumped it up to 250 since none of the chains (inc Arbitrum) produce blocks faster than that.